<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiDocs</class>
 <widget class="QDialog" name="MiDocs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/adjunto-archivo.png</normaloff>:/adjunto-archivo.png</iconset>
  </property>
  <widget class="QPushButton" name="btnBuscar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Colección de iconos/buscar 48-icon.png</normaloff>../../../Colección de iconos/buscar 48-icon.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBorrar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar Archivo</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Colección de iconos/Borrar Factura 48-icon.png</normaloff>../../../Colección de iconos/Borrar Factura 48-icon.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModificar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar Nombre</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Colección de iconos/Modificar Editar 48-icon.png</normaloff>../../../Colección de iconos/Modificar Editar 48-icon.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Colección de iconos/grabar 48-icon.png</normaloff>../../../Colección de iconos/grabar 48-icon.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSallir">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="slotArbol">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>591</width>
     <height>271</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnMonitor">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar arbol</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>140</y>
     <width>581</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <zorder>btnMonitor</zorder>
  <zorder>btnBuscar</zorder>
  <zorder>btnBorrar</zorder>
  <zorder>btnModificar</zorder>
  <zorder>btnGuardar</zorder>
  <zorder>btnSallir</zorder>
  <zorder>slotArbol</zorder>
  <zorder>listWidget</zorder>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
